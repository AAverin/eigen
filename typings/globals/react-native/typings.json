{
  "resolution": "main",
  "tree": {
    "src": "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/4c7d03b088d4e71cd9fe9c66ab7bd363049077c3/react-native/react-native.d.ts",
    "raw": "registry:dt/react-native#0.29.0+20160719232534",
    "typings": "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/4c7d03b088d4e71cd9fe9c66ab7bd363049077c3/react-native/react-native.d.ts"
  }
}
