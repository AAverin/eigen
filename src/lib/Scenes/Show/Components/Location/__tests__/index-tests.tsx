import React from "react"
import "react-native"
import * as renderer from "react-test-renderer"
import { Location } from "../index"

it("looks correct when rendererd", () => {
  const comp = renderer.create(<Location location={{}} />)
  expect(comp).toMatchSnapshot()
})
