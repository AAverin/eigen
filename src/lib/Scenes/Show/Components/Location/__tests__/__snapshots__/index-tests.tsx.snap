// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`looks correct when rendererd 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
>
  {}
</Text>
`;
